libs/feature/record/src/lib/state/mdview.actions.ts

```diff
@@ -28,6 +29,21 @@ export const loadFullMetadataFailure = createAction(
   props<{ otherError?: string; notFound?: boolean }>()
 )
 
+export const loadFeatureCatalogAttributes = createAction(
```

Comment:
```suggestion
export const loadFeatureCatalog = createAction(
```

The "attributes" part is redundant :)